dll/win32/shell32/utils.cpp

```diff
@@ -504,3 +504,29 @@ SHOpenPropSheetA(
     return SHOpenPropSheetW(pszCaptionW, ahKeys, cKeys, pclsidDefault,
                             pDataObject, pShellBrowser, pszStartPageW);
 }
+
+/*************************************************************************
+ *  Activate_RunDLL [SHELL32.105]
+ *
+ * Every parameter is unused.
+ */
+EXTERN_C
+BOOL WINAPI
+Activate_RunDLL(
+    DWORD dwUnused1,
+    LPVOID lpUnused2,
+    LPVOID lpUnused3,
+    LPVOID lpUnused4)
```

Comment:
```suggestion
    _In_ DWORD dwUnused1,
    _In_ LPVOID lpUnused2,
    _In_ LPVOID lpUnused3,
    _In_ LPVOID lpUnused4)
```

Reply:
OK yeah I don't know whether this deserves an annotation at all, seeing what this is...